modules/claims_api/app/controllers/claims_api/v0/forms/power_of_attorney_controller.rb

```diff
@@ -40,6 +40,10 @@ def submit_form_2122
           end
 
           ClaimsApi::PoaUpdater.perform_async(power_of_attorney.id)
+          if enable_vbms_access?
+            ClaimsApi::VBMSUpdater.perform_async(power_of_attorney.id,
+                                                 target_veteran.participant_id)
```

Comment:
A bit odd of formatting. 🤷‍♂️ 
```suggestion
            ClaimsApi::VBMSUpdater.perform_async(
              power_of_attorney.id, target_veteran.participant_id
            )
```